pkg/controller/hyperconverged/factory.go

```diff
@@ -130,3 +131,22 @@ func newKubeVirtTemplateValidatorForCR(cr *hcov1alpha1.HyperConverged, namespace
 		},
 	}
 }
+
+func newIMSConfig(cr *hcov1alpha1.HyperConverged, namespace string) *corev1.ConfigMap {
```

Comment:
I prefer this name to `newIMSConfigForCR` but I think it's better to not break the pattern.

Reply:
Cool. I'll change it.